client/blocks/login/login-form.jsx

```diff
@@ -78,20 +85,21 @@ export class LoginForm extends Component {
 	onSubmitForm = ( event ) => {
 		event.preventDefault();
 
-		const { password, rememberMe, usernameOrEmail } = this.state;
+		const { password, usernameOrEmail } = this.state;
+		let { rememberMe } = this.state;
```

Comment:
We should [not mutate state directly](https://facebook.github.io/react/docs/react-component.html#state), but use `this.setState()` instead.